src/libs/ajax/leonardo/Apps.ts

```diff
@@ -77,13 +77,16 @@ export const Apps = (signal) => ({
       },
     };
   },
-  listAppsV2: async (workspaceId: string): Promise<ListAppResponse[]> => {
-    const res = await fetchLeo(`api/apps/v2/${workspaceId}`, _.mergeAll([authOpts(), appIdentifier, { signal }]));
+  listAppsV2: async (workspaceId: string, labels: LeoResourceLabels = {}): Promise<ListAppResponse[]> => {
+    const res = await fetchLeo(
+      `api/apps/v2/${workspaceId}?${qs.stringify(labels)}`,
+      _.mergeAll([authOpts(), appIdentifier, { signal }])
+    );
     return res.json();
   },
-  createAppV2: (appName: string, workspaceId: string): Promise<void> => {
+  createAppV2: (appName: string, workspaceId: string, appType: string): Promise<void> => {
```

Comment:
Added `appType` as a param to `createAppV2` instead of hard-coding `CROMWELL`.

Reply:
you shouldl be able too use `AppToolLabel` as the type instead of string here. A bit of a naming mismatch, the UI calls these attributes of an app the 'label' elsewhere so it can generalize and compare resources (i.e. runtimes and apps), but `AppToolLabel` type has a one-to-one mapping with the leo `appType` schema